keyboards/westfoxtrot/aanzee/aanzee.h

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2018 westfoxtrot
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifndef AANZEE_H
+#define AANZEE_H
+
+#define ___ KC_NO
+
+#include "quantum.h"
+
+/* This a shortcut to help you visually see your layout.
+*
+* The first section contains all of the arguments representing the physical
+* layout of the board and position of the keys.
+*
+* The second converts the arguments into a two-dimensional array which
+* represents the switch matrix.
+*/
+// readability
+
+#define LAYOUT_ANSI( \
```

Comment:
This should be:
```suggestion
#define LAYOUT_ansi( \
```

Also, if this conforms to one of the community layouts, you may want to use that name, instead (though, it doesn't look like this does)